relay-server/src/services/processor/span/processing.rs

```diff
@@ -413,6 +467,73 @@ fn validate(mut span: Annotated<Span>) -> Result<Annotated<Span>, anyhow::Error>
     if let Some(tags) = tags.value_mut() {
         tags.retain(|_, value| !value.value().is_empty())
     }
+    if let Some(measurements) = measurements.value_mut() {
+        measurements.retain(|_, value| match value.value() {
+            Some(v) => !v.is_empty() && !v.value.is_empty(),
```

Comment:
Is the measurement `unit` is allowed to be empty? This condition expands to

```rust
!v.value.is_empty() && !v.unit.is_empty() && !v.value.is_empty()
```

so I think you can simplify it.

Reply:
The field can be omitted but a `null` value can't be ingested.